monitor.go

```diff
@@ -192,20 +198,30 @@ func NewMonitor(opts ...MonitorOption) *Monitor {
 }
 
 func (monitor *Monitor) monitor() {
+	changesEvents := make([]*MonitorEvent, 0, 2)
```

Comment:
Here and below: Why initialize the slice with capacity 2? It seems unlikely to me that this would make a difference. The slice’s capacity will quickly grow to the maximum needed and then stay there, I think.

Reply:
switched to

```go
make([]*MonitorEvent, 0)
```